### DISCUSSION TITLE
행복한 개발을 위한 폴더 구조는?

### DISCUSSION THREAD

**Discussion by giscus** [teamGrowing/team-blog#7] [Comments]
# 행복한 개발을 위한 폴더 구조는?

안녕하세요. 그로잉 FE팀입니다! 😀 현재, 저희는 그로잉 프로젝트 리팩토링을 앞두고 있어요. 작성한지 1-2년 된 코드였기에 본격적인 리팩토링을 시작하기 전, 사용하지 않는 파일들을 정리하고 폴더 구조를 다듬을 필요가 있었어요. 프로젝트의 폴더 구조를 잘 정의하는 것은 앞으로의 개발 효율성에 영향을 주는 중요한 요소이기도 하죠. 순간의 선택이 10년을 …

https://teamgrowing.github.io/team-blog/folder-structure/

**Comment by yeonju0110** [comment]
🩷

**Comment by metamoong** [comment]
💛
